src/components/bcros/alert/index.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <div :data-cy="`alert-display${index || index===0 ? '-' + index : ''}`" class="px-3 py-3">
+    <UIcon
+      v-if="showHeader"
+      :class="`${iconColour} mr-2 font-semibold`"
+      :name="iconName"
+      data-cy="alert-icon"
+    />
+    <span v-if="showHeader" class="font-semibold flex-auto">{{ alertHeader }}</span>
+    <button
```

Comment:
Hey Brandon can you update this to use UButton so that it picks up our common styling etc.? You will be able to remove the UIcon and it will size it similar to our other ones.
i.e. for this example it would be:
```
    <UButton
        color="primary"
        icon="i-mdi-chevron-down"
        :label="expanded ? 'Hide Details' : 'View Details'"
        trailing
        variant="ghost"
        :ui="{ icon: { base: expanded ? 'rotate-180 transition-all' : 'transition-all' } }"
        @click="expanded = !expanded"
      />
```

Reply:
Done this took a few additional changes but I believe this has better functionality now than before